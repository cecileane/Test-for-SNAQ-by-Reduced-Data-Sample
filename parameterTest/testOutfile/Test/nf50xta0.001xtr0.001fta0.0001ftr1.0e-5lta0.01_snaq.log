optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.0001,
                       xtolAbs=0.001, xtolRel=0.001.
 max number of failed proposals = 50, liktolAbs = 0.01.
rootname for files: nf50xta0.001xtr0.001fta0.0001ftr1.0e-5lta0.01_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 22059
seed: 22059 for run 1
Sat Oct  1 17:29:28 2016

 BEGIN SNaQ for run 1, seed 22059 and hmax 1
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 61 (failures: 50) with -loglik=126.74251 and ht_min=[0.2856,1.51215,0.51916,10.0,0.48942]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 17		 17		 0	 |	 0		 0		 --
mvtarget	 16		 16		 0	 |	 0		 0		 --
chdir		 6		 6		 2	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 20		 15		 1	 |	 5		 5		 --
Total		 61		 56		 4	 |	 5		 5		 4
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 0.8

 FINISHED SNaQ for run 1, -loglik of best 126.74250982222948
(((2,(1)#H7:::0.7143961297286794):1.5121490165932967,4):0.48941967235201217,3,(5,(6,#H7:::0.2856038702713206):10.0):0.5191597703513875);
---------------------
seed: 91660 for run 2
Sat Oct  1 17:29:33 2016

 BEGIN SNaQ for run 2, seed 91660 and hmax 1
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 74 (failures: 50) with -loglik=127.5115 and ht_min=[0.54608,0.36548,0.5186,2.23709,10.0,0.0,0.0]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 23		 23		 0	 |	 0		 0		 --
mvtarget	 20		 20		 1	 |	 0		 0		 --
chdir		 5		 5		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 25		 14		 1	 |	 2		 2		 --
Total		 74		 63		 4	 |	 2		 2		 2
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 2, -loglik of best 127.51150018542359
((((5,#H7:0.3654803929584435::0.45391557744231315):10.0,6):2.237093682240734,1):0.5185998151002711,2,((4,3):0.0)#H7:0.0::0.5460844225576869);
---------------------

Sat Oct  1 17:29:37 2016
MaxNet is (((2,(1)#H7:::0.7143961297286794):1.5121490165932967,4):0.48941967235201217,3,(5,(6,#H7:::0.2856038702713206):10.0):0.5191597703513875); 
with -loglik 126.74250982222948
