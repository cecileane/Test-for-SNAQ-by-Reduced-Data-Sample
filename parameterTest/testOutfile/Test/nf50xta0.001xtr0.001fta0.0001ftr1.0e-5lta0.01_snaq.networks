(((2,(1)#H7:::0.7143961297286794):1.5121490165932967,4):0.48941967235201217,3,(5,(6,#H7:::0.2856038702713206):10.0):0.5191597703513875);, with -loglik 126.74250982222948 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(((2,(1,#H7:::0.31315830161770797):3.5336453219787534):0.6670598244739346,4):0.4629672509852351,3,(5,(6)#H7:::0.686841698382292):1.7039912490923628);, with -loglik 128.3199308258511
((4)#H7:::0.6088148687812458,3,(5,(6,(1,(#H7:::0.3911851312187542,2):2.941203471007553):1.1938368189578403):0.2196528351651028):0.02272145326146192);, with -loglik 172.3127309496454
((#H7:::0.42731007432415735,4):0.8968465302291679,3,(5,(6,(1,(2)#H7:::0.5726899256758426):3.4608695264841076):0.20939685606565306):0.05318364338939011);, with -loglik 204.75687111715416
((2,(1,((5,#H7:::0.48842950831118237):9.75,6):1.1336093897414747):0.0):0.0,4,(3)#H7:::0.5115704916888176);, with -loglik 230.44947104564812
(((2,(1,((5)#H7:::0.5052500506317337,6):2.925229786624609):0.035208313910979866):0.20097099449469996,4):1.3820560230915029,3,#H7:::0.49474994936826633);, with -loglik 233.662916006502
