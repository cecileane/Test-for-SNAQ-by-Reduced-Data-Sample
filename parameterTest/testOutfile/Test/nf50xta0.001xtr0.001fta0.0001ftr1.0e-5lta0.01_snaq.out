(((2,(1)#H7:::0.7143961297286794):1.5121490165932967,4):0.48941967235201217,3,(5,(6,#H7:::0.2856038702713206):10.0):0.5191597703513875); -Ploglik = 126.74250982222948
 Dendroscope: (((2,(1)#H7):1.5121490165932967,4):0.48941967235201217,3,(5,(6,#H7):10.0):0.5191597703513875);
 Elapsed time: 9.456467271 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (((2,(1)#H7:::0.7143961297286794):1.5121490165932967,4):0.48941967235201217,3,(5,(6,#H7:::0.2856038702713206):10.0):0.5191597703513875);, with -loglik 126.74250982222948
 ((((5,#H7:0.3654803929584435::0.45391557744231315):10.0,6):2.237093682240734,1):0.5185998151002711,2,((4,3):0.0)#H7:0.0::0.5460844225576869);, with -loglik 127.51150018542359
-------
